krank.cabal

```diff
@@ -31,7 +31,8 @@ library
                        , req >= 2.1.0 && < 2.2
                        , text >= 1.2.3 && < 1.3
                        , unordered-containers >= 0.2.10 && < 0.2.11
-  hs-source-dirs:      src
+                       , utf8-string >= 1.0.1.1 && < 1.1
+  Hs-source-dirs:      src
```

Comment:
Why the capitalization here?

Reply:
because it looks nicer? :)

I must have done that without realizing it